Docs/11 DOM/README.md

```diff
@@ -1,160 +1,337 @@
+
+```fsharp
+module DynamicViews =
+  open FSharp.Data.Adaptive
+  open Fun.Blazor
+
+  let adaptiveNode initialAge =
+    adaptiview () {
+      let! age, setAge = cval(10).WithSetter()
+
+      section {
+        $"Age: {age}"
+        br
+
+        input {
+          type' "number"
+          value age
+          onchange (fun event -> unbox<string> event.Value |> int |> setAge)
         }
-    }        
+      }
+    }
 ```
 
-For DOM element/component, you can do:
+## Stores and Observables
+
+> Note: This requires the [Fun.Blazor.Reactive] package
+
+### Observables
+
+> Note: The examples here use the [FSharp.Control.Reactive] package due the functions it provides to help, but it is not required, any `IObservable<T>` works with Fun.Blazor
 
-Without any bindings you can:
+If you have observable information around it is quite simple to hook it up with Fun.Blazor by using the `html.watch` API
 
 ```fsharp
-let demo =
-    html.blazor (ComponentAttrBuilder<MudPaper>()
-        .Add((fun x -> x.Elevation), 10)    
-        .Add((fun x -> x.Outlined), true)    
-    )
+module ReactiveViews =
+  open System
+  open Fun.Blazor
+  open FSharp.Control.Reactive
+
+  let obs = Observable.interval (TimeSpan.FromSeconds(1.))
+
+  article {
+
+    h1 { "My View" }
+    p {
+      html.watch(obs, (fun num -> fragment { $"Number: {num}" }), 0)
+    }
+  }
 ```
 
-With auto generated bindings:
+### Stores
+
+Another popular way to handle state changes in the frontend ecosystem is the usage of stores which are containers that keep data around for you. they are very similar to observables but they provide a simpler API to work with. Stores are also useful to bind dynamic attributes
 
 ```fsharp
-open Fun.Blazor
-open Fun.Blazor.Operators
-open MudBlazor
-
-type MudTable'<'T> with
-
-    [<CustomOperation("HeaderAndRow")>]
-    member this.HeaderAndRow(render: AttrRenderFragment, mappers: (NodeRenderFragment * ('T -> NodeRenderFragment)) seq) =
-        let headers = mappers |> Seq.map fst
-        let render = this.HeaderContent(render, html.fragment headers)
-        this.RowTemplate(render, (fun row -> html.inject (row, (fun () -> mappers |> Seq.map (snd >> fun fn -> fn row) |> html.fragment))))
-
-    [<CustomOperation("withDefaultSettings")>]
-    member inline _.withDefaultSettings([<InlineIfLambda>] render: AttrRenderFragment) =
-        render
-        ==> MudTable'() {
-            Hover true
-            FixedHeader true
-            HorizontalScrollbar true
-            Breakpoint Breakpoint.None
-            asAttrRenderFragment // with this feature we can have a better coding experience
-        }
+module ReactiveViews =
+  open Fun.Blazor
+
+  let store: IStore<int> = new Store<int>(initialAge)
+
+  let view() =
+    let storeNode =
+      html.watch(store, (fun num -> fragment { $"Store: {num}" }))
+
+    article {
+      html.bind("data-current", store)
```

Comment:
Not all attribute can be bind with this. It is used to bind a csharp bindable property, which actually has two attributes under the hood, like Value and ValueChanged

Reply:
I'll remove it from this example for the moment it should not affect the rest of the code example